server/email-templates/email-templates.es6

```diff
@@ -0,0 +1,270 @@
+'use strict';
+
+const moment = require('moment');
+const vcapConstants = require('../vcap-constants.es6');
+const util = require('../util.es6');
+const email = {};
+
+email.noncommercialApplicationSubmittedConfirmation = application => {
```

Comment:
I believe it would be cleaner to have a directory of email templates so that someone less familiar with the site could jump in the relevant file.